### PR TITLE
MINOR FIX :  Student Exam Service Exception Add

### PR DESCRIPTION
**Pull request by tharanisantosh**

### CODE REVIEW

File: OtripleS.Web.Api/Models/StudentExams/Exceptions/FailedStudentExamServiceException.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+using System;
+using Xeptions;
+
+namespace OtripleS.Web.Api.Models.StudentExams.Exceptions
+{
+    public class FailedStudentExamServiceException : Xeption
+    {
+        public FailedStudentExamServiceException(Exception innerException)
+            : base(message: "Failed student exam service error occured.",innerException)
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api/Models/StudentExams/Exceptions/FailedStudentExamServiceException.cs:14]: contact support.","<space>innerException

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/StudentExams/StudentExamServiceTests.Validations.Remove.cs [hunk 2]
```diff
@@ -104,7 +104,10 @@ public async Task ShouldThrowServiceExceptionOnDeleteWhenExceptionOccursAndLogIt
             Guid inputStudentExamId = randomStudentExamId;
             var exception = new Exception();
 
-            var expectedStudentExamServiceException = new StudentExamServiceException(exception);
+            var failedStudentExamServiceException =
+                new FailedStudentExamServiceException(exception);
+
+            var expectedStudentExamServiceException = new StudentExamServiceException(failedStudentExamServiceException);
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/StudentExams/StudentExamServiceTests.Validations.Remove.cs:110]: can be broken to two lines

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/StudentExams/StudentExamServiceTests.Validations.Remove.cs (hunk 2 above)
Comment (ShriHumrudha) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/StudentExams/StudentExamServiceTests.Validations.Remove.cs:110]: @tharanisantosh this is pending

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/StudentExams/StudentExamServiceTests.Validations.Remove.cs (hunk 2 above)
Comment (tharanisantosh) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/StudentExams/StudentExamServiceTests.Validations.Remove.cs]: Comment Resolved.

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**